### PR TITLE
feat: add --controls-backdrop-color CSS var to allow changing the backdrop color

### PR DESCRIPTION
**Pull request by gkatsev**
~"controls shade color" is sort of a placeholder to kick off a discussion for a better name.~

You can try it out on the MuxPlayer nextjs test page.

### CODE REVIEW

**Comment by luwes** [comment]
I've brainstormed about this before, what do y'all think of `backdrop`?
see https://developer.mozilla.org/en-US/docs/Web/CSS/::backdrop

**Comment by gkatsev** [comment]
backdrop might be good. should it be `--backdrop-color` or `--controls-backdrop-color`? Or maybe `--media-backdrop-color`?

**Comment by luwes** [comment]
`--controls-backdrop-color` seems more accurate to me. 


`--media-backdrop-color` could mean for transparent video the color behind the video element
https://ataylor32.github.io/demo-html5-transparent-video/

**Comment by gkatsev** [comment]
New design doesn't have a full player backdrop, so, going to close this, and we can revisit when we start targeting that design update.

File: packages/mux-player/README.md [hunk 1]
```diff
@@ -198,6 +198,18 @@ Instead, some components expose **parts** that can be targeted with the [CSS par
...
+
+We expose a CSS variable to set the controls backdrop color.
+
+```css
+mux-player {
+  --controls-backdrop-color: rgb(0 0 0 / 0%);
+}
+```
+
+Turning this off completely as implications on the accessibility of the controls as they may not meet [the contrast ratio requirements for WCAG 2.1](https://www.w3.org/TR/WCAG/#contrast-minimum) without it.
```
Comment (luwes) [inline packages/mux-player/README.md:211]:
```suggestion
Turning this off completely has implications on the accessibility of the controls as they may not meet [the contrast ratio requirements for WCAG 2.1](https://www.w3.org/TR/WCAG/#contrast-minimum) without it.
```

**Comment by luwes** [comment]
was wondering if we should check if it's worth to open up adding the background prop.
would be nice for gradients and such.
https://github.com/muxinc/elements/pull/389

the only thing that is not ideal is that it had to be a layer lower than vertical-layer so on gestures-layer...

File: packages/mux-player/README.md [hunk 2]
```diff
@@ -198,6 +198,18 @@ Instead, some components expose **parts** that can be targeted with the [CSS par
...
+
+We expose a CSS variable to set the controls backdrop color.
+
+```css
+mux-player {
+  --controls-backdrop-color: rgb(0 0 0 / 0%);
+}
+```
+
+Turning this off completely has implications on the accessibility of the controls as they may not meet [the contrast ratio requirements for WCAG 2.1](https://www.w3.org/TR/WCAG/#contrast-minimum) without it.
```
Comment (cjpillsbury) [inline packages/mux-player/README.md:211]: suggestion(non-blocking)
> Turning this off completely or adjusting it significantly can effect accessibility considerations and compliance for [the contrast ratio requirements ...

(or similar)
Also, in my meeting w/the a11y audit company, they let me know that https://www.w3.org/TR/WCAG22/ just happened (which they tested against, iirc)

**Review by cjpillsbury — APPROVED** [review]
LGTMe. Will explore a followup on the example app in my refactor effort, per conversation.

**Comment by cjpillsbury** [comment]
> the only thing that is not ideal is that it had to be a layer lower than vertical-layer so on gestures-layer...

@luwes Won't this be incompatible with your recent change to rely on `display: none` for disabling gestures?

File: packages/mux-player/README.md (hunk 2 above)
Comment (gkatsev) [reply packages/mux-player/README.md:211]: Looks like 2.2 is still in CR and not a TR just yet, but I do think it'll be officially official 🔜

**Comment by luwes** [comment]
@cjpillsbury no the gestures-layer itself which itself is a slot element now will not be display none. only the elements within

**Review by luwes — APPROVED** [review]
👍 we can revisit gradients later if there is interest

**Pull request merged**